<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Command line suite" duration-ms="35135" started-at="2018-09-17T12:45:19Z" finished-at="2018-09-17T12:45:54Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="35135" started-at="2018-09-17T12:45:19Z" finished-at="2018-09-17T12:45:54Z">
      <class name="com.qa.Homepage.FailedTestCase">
        <test-method status="FAIL" signature="test()[pri:0, instance:com.qa.Homepage.FailedTestCase@3551a94]" name="test" duration-ms="18" started-at="2018-09-17T14:45:19Z" finished-at="2018-09-17T14:45:19Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at com.qa.Homepage.FailedTestCase.test(FailedTestCase.java:11)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- com.qa.Homepage.FailedTestCase -->
      <class name="com.qa.Homepage.LoginTest">
        <test-method status="PASS" signature="Setup()[pri:0, instance:com.qa.Homepage.LoginTest@531be3c5]" name="Setup" is-config="true" duration-ms="8789" started-at="2018-09-17T14:45:19Z" finished-at="2018-09-17T14:45:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="PASS" signature="ContactTest()[pri:0, instance:com.qa.Homepage.LoginTest@531be3c5]" name="ContactTest" duration-ms="8667" started-at="2018-09-17T14:45:28Z" finished-at="2018-09-17T14:45:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ContactTest -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:com.qa.Homepage.LoginTest@531be3c5]" name="TearDown" is-config="true" duration-ms="144" started-at="2018-09-17T14:45:36Z" finished-at="2018-09-17T14:45:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method status="PASS" signature="Setup()[pri:0, instance:com.qa.Homepage.LoginTest@531be3c5]" name="Setup" is-config="true" duration-ms="7702" started-at="2018-09-17T14:45:37Z" finished-at="2018-09-17T14:45:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="PASS" signature="VerifyLogin()[pri:0, instance:com.qa.Homepage.LoginTest@531be3c5]" name="VerifyLogin" duration-ms="9587" started-at="2018-09-17T14:45:44Z" finished-at="2018-09-17T14:45:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLogin -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:com.qa.Homepage.LoginTest@531be3c5]" name="TearDown" is-config="true" duration-ms="180" started-at="2018-09-17T14:45:54Z" finished-at="2018-09-17T14:45:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- com.qa.Homepage.LoginTest -->
    </test> <!-- Command line test -->
  </suite> <!-- Command line suite -->
</testng-results>
